db-migrations/000012_adding_indexes_on_aws_resource_relationship.up.sql

```diff
@@ -0,0 +1,7 @@
+-- Adding indexes on aws_resource_relationship
+BEGIN;
+
+CREATE UNIQUE INDEX IF NOT EXISTS aws_resource_relationship_idx_no_after ON aws_resource_relationship (not_before, arn_id, related_arn_id) WHERE not_after IS NULL;
```

Comment:
question: do we have performance concerns about adding this index without doing so `CONCURRENTLY`?

Reply:
Not really, table is empty currently.